pkg/chain/evmclient.go

```diff
@@ -19,17 +19,23 @@ import (
 // evmClient expends the base EVM client by splitting batch calls into sub-batches
 type evmClient struct {
 	*ethclient.Client
+	chHead    chan types.BlockKey
 	rpcClient *rpc.Client
 	batchSize int
 }
 
 // NewEVMClient is the constructor of evmClient
 func NewEVMClient(client *rpc.Client, batchSize int) types.EVMClient {
-	return &evmClient{
+	c := &evmClient{
 		Client:    ethclient.NewClient(client),
+		chHead:    make(chan types.BlockKey, 1000),
```

Comment:
any reason to not have it 1 length as we had before

Reply:
Same as above